### DISCUSSION TITLE
How do I run Konva? http://localhost:1234 responds with 404 page not found.

### DISCUSSION THREAD

**Discussion by adnanlah** [konvajs/konva#1472] [Q&A]
Hello,

I wanted to fork konva to try to make some changes but I didn't know how to start. I ran `yarn start` and it ran the server on `localhost:1234` but the server responds with 404! I tried `http://localhost:1234/test/runner.html` and it's the same thing.

What should I do?

**Comment by lavrton** [comment]
You should use http://localhost:1234/unit-tests.html. I updated readme. Please try it.
